### PR TITLE
feat: service access auth and refactor project list page

### PR DESCRIPTION
**Pull request by codding-y**

### CODE REVIEW

File: src/Services/MASA.PM.Service.Admin/Infrastructure/Entities/Cluster.cs [hunk 1]
```diff
@@ -15,5 +15,16 @@ public class Cluster : BaseEntity<int, Guid>
         [Required(ErrorMessage = "Cluster description is required")]
         [StringLength(255, ErrorMessage = "Cluster description length must be less than 255")]
         public string Description { get; set; } = "";
+
+        public Cluster()
```
Comment (doddgu) [inline src/Services/MASA.PM.Service.Admin/Infrastructure/Entities/Cluster.cs:19]: Maybe private enough.

File: src/ApiGateways/Caller/MASA.PM.Caller/_imports.cs [hunk 2]
```diff
@@ -2,5 +2,7 @@
 // Licensed under the Apache License. See LICENSE.txt in the project root for license information.
```
Comment (doddgu) [inline src/ApiGateways/Caller/MASA.PM.Caller/_imports.cs:2]: imports -> Imports

File: src/Web/MASA.PM.Web.Admin/Pages/Home/Team.razor.cs [hunk 3]
```diff
@@ -28,94 +28,75 @@ public partial class Team
...
         private UserPortraitModel _userInfo = new();
         private ProjectModal? _projectModal;
         private AppModal? _appModal;
+        private ProjectList? _projectListComponent;
 
         protected override void OnInitialized()
         {
-            NavigationManager.LocationChanged += HandleLocationChanged;
-            base.OnInitialized();
+            //NavigationManager.LocationChanged += HandleLocationChanged;
```
Comment (doddgu) [inline src/Web/MASA.PM.Web.Admin/Pages/Home/Team.razor.cs:49]: ?

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Services/MASA.PM.Service.Admin/Infrastructure/Entities/Cluster.cs (hunk 1 above)
Comment (codding-y) [reply src/Services/MASA.PM.Service.Admin/Infrastructure/Entities/Cluster.cs:19]: must be public, because use in other code

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**